cypress-io/cypress
# feat: display an activity indicator in the CLI output when uploading artifacts to cypress cloud

- Closes [#28239](https://github.com/cypress-io/cypress/issues/28239)

### Additional details

When uploading large artifacts to the cloud, we need a keep alive signal for CI systems, as well as an indicator for users when running in record mode locally.

### Steps to test

1. Throttle network connections using charles proxy or similar (alternatively, select a project that generates a large test replay database or video)
2. Run cypress in record mode
3. Verify the upload indicator progresses

### How has the user experience changed?
Before:
<img width="820" alt="Screen Shot 2023-11-02 at 12 31 40 PM" src="https://github.com/cypress-io/cypress/assets/142877/dbb7760b-acb6-4800-994e-af582307bd4e">

After:
![out_delayed](https://github.com/cypress-io/cypress/assets/142877/4e4911af-c734-4836-90eb-0a0c9e9fa530)


### PR Tasks


- [NA] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [NA] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
